content/dictionary_pages/letter_1.md

```diff
@@ -0,0 +1,35 @@
+title: А
+slug: letter-1
+category: dictionary
+tags: [_nav, _category_header]
+---
+
+**Абсентеизм (от лат. absentia – отсутствие)** – отсутствие работника на работе, невыход на работу без уважительной причины
```

Comment:
Although it's okay, our convention is to use `__` for bold text, not `**`. Find and replace all the occurrences.